pkg/benchmarks/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+module.exports = {
+  env: {
+    commonjs: true
```

Comment:
this isn't actually correct.  peg is expected to run in browsers and free engines, not just in node.

this disables eslint's ability to warn you when you're assuming node tools exist, which often won't.

Reply:
ok